autoconnect/autoconnect-settings/src/lib.rs

```diff
@@ -130,7 +130,7 @@ impl Default for Settings {
             // Matches the legacy value
             statsd_label: "autopush".to_owned(),
             statsd_port: 8125,
-            db_dsn: None,
+            db_dsn: Some("grpc://localhost:8086".to_string()),
```

Comment:
I think this should continue to be `None` (both here and autoendpoint) as I'd rather a misconfigured autopush fail fast with an error rather than attempting to use this url.

Instead I think `test_settings` should set this `db_dsn` value itself. I'm not sure autoendpoint has any equivalent or needs this, anyway

Reply:
Good point. Updated 👍